styles.css

```diff
@@ -186,7 +228,8 @@ div.canopy h5 {
   line-height: 16px;
   text-align: left;
   align-items: center;
-  vertical-align: center;
+
+  /* vertical-align: center; */
```

Comment:
- [ Required ] I would like to suggest that you delete this commented line of code, this is because commented code makes a project look unprofessional and is widely considered as a sign of code that is not clean. Please rectify this.